app/models/health/vaccination.rb

```diff
@@ -29,8 +29,13 @@ class Vaccination < EpicBase
     phi_attr :dob, Phi::Date, 'Date of birth of patient'
     phi_attr :ssn, Phi::Ssn, 'Social security number of patient'
     phi_attr :existed_previously, Phi::NeedsReview
+    phi_attr :data_source_id, Phi::SmallPopulation, "Source of data (may identify provider)"
 
-    has_one :patient, primary_key: :medicaid_id, foreign_key: :medicaid_id
+    has_one :patient, **epic_assoc(
+      model: :patient,
+      primary_key: :medicaid_id,
+      foreign_key: :medicaid_id,
+    )
```

Comment:
Is `data_source_id` really needed here?  Won't patients have epic patients, but ideally they'd be merged if they share a Medicaid ID?